src/components/ContactInfo/ContactInfo.vue

```diff
@@ -16,7 +16,7 @@
       <!-- Email Address -->
       <v-flex xs4>
         <label class="d-flex flex-wrap">
-          <span class="subtitle mr-2">Email Address</span>
+          <span class="subtitle text-body-3 mr-2">Email Address</span>
```

Comment:
text-body-3 is set to
````
  .text-body-3 {
    font-family: inherit;
    font-size: 0.875rem !important;
    line-height: 1.375rem !important;
  }

but in the design doc it has 14px. What font-size should we use?
````

Reply:
14px converted to rem is .875.

As a rule of thumb, we've traditionally used rem for font-sizes.

Reply:
https://www.google.com/search?q=why+use+rem+instead+of+px

Reply:
👍